docs/tutorials/tutorial-delete-data.md

```diff
@@ -97,35 +97,29 @@ $ ls -l1 var/druid/segments/deletion-tutorial/
 
 Let's disable some segments by their segmentID. This will again mark the segments as "unused", but not remove them from deep storage. You can see the full segmentID for a segment from UI as explained below.
```

Comment:
```suggestion
Let's disable some segments by their segmentID. This will again mark the segments as "unused", but not remove them from deep storage. You can see the full segmentID for a segment using the web console.
```